bin/desi_night_qa

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE.rst file for Copyright information
+#
+# -*- coding: utf-8 -*-
+
+"""
+This script generates the $DESI_ROOT/spectro/redux/nightqa/{NIGHT}/nightqa-{NIGHT}.html page, and related products, once all tile-qa*fits are done.
+"""
+
+
+import os,sys
+import argparse
+from desiutil.log import get_logger
+from desispec.io import specprod_root
+from desispec.night_qa import (
+    get_nightqa_outfns,
+    get_survey_night_expids,
+    get_dark_night_expid,
+    create_dark_mp4,
+    create_badcol_mp4,
+    create_ctedet_mp4,
+    create_sframesky_mp4,
+    create_tileqa_mp4,
+    write_nightqa_html,
+)
+
+def parse(options=None):
+    parser = argparse.ArgumentParser(
+                description="Generate $DESI_ROOT/spectro/redux/nightqa/{NIGHT}/nightqa-{NIGHT}.html page, and related products")
+    parser.add_argument("-p", "--prod", type = str, default = None, required = False,
+                        help = "Path to input reduction, e.g. /global/cfs/cdirs/desi/spectro/redux/blanc/,  or simply prod version, like blanc, but requires env. variable DESI_SPECTRO_REDUX. Default is $DESI_SPECTRO_REDUX/$SPECPROD.")
+    parser.add_argument("-n","--night", type = int, default = None, required = True,
+                        help = "night to process. ex: 20211128")
+    parser.add_argument("-o", "--outdir", type = str, default = None, required = False,
+                        help = "Path to ouput folder, default is the input prod directory. Files written in {prod}/nightqa/{night}; several files will be created there")
+    parser.add_argument("--recompute", action = "store_true",
+                        help = "recompute (i.e. overwrite args.outfile if already existing")
+    parser.add_argument("--css", type = str, default = None, required = False,
+                        help = "html formatting css file; default to {prod}/nightqa/nightqa.css")
```

Comment:
Let's put the CSS file in `py/desispec/data/qa/nightqa.css` so that desispec remains self-contained for what is needed to run this.  Use `pkg_resources.resource_filename('desispec', 'data/qa/nightqa.css')` to find it and then copy that into the output directory.  If you want to leave an option to specify a different CSS ok.

I think we *do* want to copy it into the output directory every time so that if we make an update to the CSS it gets automatically propagated and the old CSS doesn't block the new one from getting installed into the output dir.

Reply:
done